src/fu-engine.c

```diff
@@ -1401,6 +1402,20 @@ fu_engine_get_report_metadata (FuEngine *self, GError **error)
 	if (!fu_engine_get_report_metadata_os_release (hash, error))
 		return NULL;
 
+	/* DMI data */
+	tmp = fu_hwids_get_value (self->hwids, FU_HWIDS_KEY_PRODUCT_NAME);
+	if (tmp != NULL)
+		g_hash_table_insert (hash, g_strdup ("HostProduct"), g_strdup (tmp));
+	tmp = fu_hwids_get_value (self->hwids, FU_HWIDS_KEY_FAMILY);
+	if (tmp != NULL)
+		g_hash_table_insert (hash, g_strdup ("HostFamily"), g_strdup (tmp));
+	tmp = fu_hwids_get_value (self->hwids, FU_HWIDS_KEY_PRODUCT_SKU);
+	if (tmp != NULL)
+		g_hash_table_insert (hash, g_strdup ("HostSku"), g_strdup (tmp));
+	tmp = fu_hwids_get_value (self->hwids, FU_HWIDS_KEY_MANUFACTURER);
+	if (tmp != NULL)
+		g_hash_table_insert (hash, g_strdup ("HostVendor"), g_strdup (tmp));
+
```

Comment:
Sending these up as part of update reports does open up some interesting things in terms of tying SMBIOS strings to what they are actually used for in UEFI capsule updates.  The concept of a "system" on LVFS and the types of updates it supports and encompasses is possible with this.

Reply:
Yes, it's a bit of _leakage_ for things like FMP, but it's probably going to be super useful for debugging.